src/router.js

```diff
@@ -63,28 +57,26 @@ const checkAndSubscribeToLobby = async (to) => {
 const routes = [
   {
     path: '/',
-    name: 'Home',
-    component: HomeView,
+    name: ROUTE_NAME_HOME,
+    component: () => import('@/routes/home/HomeView.vue'),
     beforeEnter: mustBeAuthenticated,
   },
   {
     path: '/login/:lobbyRedirectId?',
     name: ROUTE_NAME_LOGIN,
-    component: LoginView,
+    component: () => import('@/routes/login/LoginView.vue'),
     meta: {
       hideNavigation: true,
     },
   },
   {
     path: '/signup',
     name: ROUTE_NAME_SIGNUP,
-    component: LoginView,
+    component: () => import('@/routes/login/LoginView.vue'),
     meta: {
       hideNavigation: true,
     },
   },
-  // This route is just a passthrough to make sure the user is fully logged out before putting
```

Comment:
I’m going to assume you used AI a bit to help with this task as it tends to remove comments unless you specify that the model keep them, and that’s totally fine, but we need these comments to be here.

Reply:
Yea, little bit. Just for refactoring and little bit for the understanding about the code

Reply:
@ar1701 if you would still like to complete this issue I believe all that remains is re-adding the previously deleted comments

Reply:
Ok...i will do it by tonight

Reply:
@itsalaidbacklife I have added the previous comments and did some modifications . Please verify once

Reply:
Just make sure you push it up, I don’t see those changes yet. Looking forward to it!